test/features/support/fixtures.js

```diff
@@ -6,5 +6,8 @@ module.exports = {
     sourceUrl: 'http://example.com',
     rating: '4',
     deadline: '20/10/2016',
+    deadlineDate: null,
+    applicationDate: null,
+    interviewDate: null,
```

Comment:
maybe it is worth adding a missing `status` field?
date for status should not be null